specification/containerservice/resource-manager/Microsoft.ContainerService/aks/preview/2023-09-02-preview/managedClusters.json

```diff
@@ -6873,6 +6877,16 @@
         }
       }
     },
+    "AgentPoolArtifactStreamingProfile": {
+      "type": "object",
+      "properties": {
+        "enabled": {
+          "type": "boolean",
+          "title": "Whether to enable artifact streaming on an agent pool. Note that corresponding container images need have artifact streaming enabled through ACR for faster pod starts.",
```

Comment:
minor: I think you can drop this section here: "Note that corresponding container images need have artifact streaming enabled through ACR for faster pod starts.", because you basically say it below.

Then below, apply the change I mention in the other comment.

Reply:
That way you reduce duplication in the comments. Other than that, looks great.

Reply:
Removed the `title` section. Hoping that the presence of a title is not a requirement.